.stylelintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": ["stylelint-config-standard"],
+    "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
+    "rules": {
+      "at-rule-no-unknown": null,
+      "scss/at-rule-no-unknown": true,
+      "csstree/validator": true
+    },
+    "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css"]
+  }
```

Comment:
kindly set up Linter correctly for this repository. The Linter result should be green. Please check this [LINK](https://github.com/microverseinc/linters-config) for help.
### Your Linter result:
![linter](https://user-images.githubusercontent.com/10332499/148194999-72870f89-2e32-43ca-af9f-40765e7e0ece.PNG)